pkg/shell/index.html

```diff
@@ -8,9 +8,6 @@
     <link href="../../static/branding.css" rel="stylesheet" />
     <script src="../base1/cockpit.js"></script>
     <script src="../manifests.js"></script>
-    <!-- HACK: C bridge loads translations via glob and Python via manifest.js -->
-    <script src="../*/po.manifest.js"></script>
-    <script src="../*/po.js"></script>
```

Comment:
As we beiboot in cockpit client and other situations I suppose there is no way an old bridge server new shell code. So this seems fine.

Reply:
Yes, that was my thought as well. The debs and rpm ensure that cockpit-{bridge,system} have the same version, and Arch doesn't even split the package.